roles/openshift_logging_eventrouter/tasks/install_eventrouter.yaml

```diff
@@ -0,0 +1,101 @@
+# initial checks
+- fail:
```

Comment:
I was advised by @ewolinetz the preferred patter going forward is to use 'assert' instead of fail